File: SDKs/amazonaws.com/models.lex.v2/2020-08-07/python/src/sdk/models/shared/slotdefaultvaluespecification.py
import dataclasses
from dataclasses_json import dataclass_json
from sdk import utils
from ..shared import slotdefaultvalue as shared_slotdefaultvalue


@dataclass_json
@dataclasses.dataclass
class SlotDefaultValueSpecification:
    r"""SlotDefaultValueSpecification
    Defines a list of values that Amazon Lex should use as the default value for a slot.
    """
    
    default_value_list: list[shared_slotdefaultvalue.SlotDefaultValue] = dataclasses.field(metadata={'dataclasses_json': { 'letter_case': utils.field_name('defaultValueList') }})
    

File: SDKs/amazonaws.com/models.lex.v2/2020-08-07/java/lib/src/main/java/openapisdk/models/shared/SlotDefaultValueSpecification.java
package openapisdk.models.shared;

import com.fasterxml.jackson.annotation.JsonProperty;

/**
 * SlotDefaultValueSpecification
 * Defines a list of values that Amazon Lex should use as the default value for a slot.
**/
public class SlotDefaultValueSpecification {
    @JsonProperty("defaultValueList")
    public SlotDefaultValue[] defaultValueList;
    public SlotDefaultValueSpecification withDefaultValueList(SlotDefaultValue[] defaultValueList) {
        this.defaultValueList = defaultValueList;
        return this;
    }
}